### ISSUE TITLE
Session Cookie does not get updated when changing timeout setting of a user

### ISSUE THREAD

**Issue by kmerz** [Graylog2/graylog2-server#12312]
## Expected Behavior
If I change the timeout setting of a user the current session cookie of a user should get either updated or invalidated.
If a admin is changing the setting from "Does not timeout" to "8 hours" the admin needs to trust the user that he is
logging out so he gets a new session cookie with a new timeout.

## Current Behavior
After changing the timeout setting from e.g 1 hour to 1 minute the session is still active for 1 hour.

## Possible Solution
Either update or invalidate the current user session so he either gets a new session cookie with updated timeout
or he needs to login anew to get a new session cookie.

## Steps to Reproduce (for bugs)
1. Log in as a admin user (not admin himself)
2. Change your setting from anything bigger than 1 minute to 1 minute
3. Wait for 1 minute 
4. Test if you are still logged in.

## Context
If a admin is changing the setting from "Does not timeout" to "8 hours" the admin needs to trust the user that he is
logging out so he gets a new session cookie with a new timeout.

## Your Environment
* Graylog Version: Graylog 4.3.0-beta.2+a61a34c

**Comment by linuspahl** [comment]
@boosty Yes, it behaved the same way in previous version (pre 4.3). The new session timeout will only be applied after the user logs in again.

**Comment by boosty** [comment]
@linuspahl Thanks! Then I would say that we should not rush to fix this for 4.3.
